### PR TITLE
Fast start

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: core/src/main/java/org/mapfish/print/VersionInfo.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ private Attributes getAttributes() {
                 }
             }
         } catch (FileNotFoundException e) {
-            LOGGER.warn("Cannot find servlet manifest", e);
```
Comment (sbrunner) [inline core/src/main/java/org/mapfish/print/VersionInfo.java]: :-)

File: core/src/main/java/org/mapfish/print/processor/map/scalebar/ScalebarGraphic.java [hunk 2]
```diff
@@ -249,7 +249,6 @@ protected static Dimension getSize(final ScalebarAttributeValues scalebarParams,
             final ScaleBarRenderSettings settings, final Dimension maxLabelSize) {
         final float width;
         final float height;
-        final double angle = Math.abs(Math.toRadians(scalebarParams.labelRotation));
```
Comment (sbrunner) [inline core/src/main/java/org/mapfish/print/processor/map/scalebar/ScalebarGraphic.java]: :-)

**Comment by sbrunner** [comment]
```
...ExamplesTest - Run example 'printtiledwms' (simpleReport.jrxml)
...ExamplesTest - Run example 'printtiledwms' (requestData-bbox-tiled-wms1_1_1.json)
```
=> I think that the message should be move here:
https://github.com/mapfish/mapfish-print/pull/539/commits/117e9588e2114e43ee67c73039607bf992f287bd#diff-007ba379756e0f55b7eaf88cb70d2cf4L192

**Comment by pvalsecc** [comment]
>  I think that the message should be move here

OK, done

**Pull request merged**